core/native/src/main/scala/bobcats/evp.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package bobcats
+
+import scala.scalanative.unsafe._
+import scala.annotation.nowarn
+
+@extern
+@link("crypto")
+@nowarn
+private[bobcats] object evp {
+
+  type EVD_MD_CTX
+  type EVD_MD
+  type ENGINE
+
+  final val EVP_MAX_MD_SIZE = 64
+
+  /**
+   * See [[https://www.openssl.org/docs/man3.1/man3/OpenSSL_add_all_algorithms.html]]
+   */
+  def OpenSSL_add_all_algorithms(): Unit = extern
```

Comment:
This doesn't seem to be needed for the current algorithms, but leaving it there for now, until I get everything else to pass.